packages/cli-core/API.md

```diff
@@ -38,6 +38,7 @@ export enum LogLevel {
 // @public
 export class Printer {
     constructor(minimumLogLevel: LogLevel, stdout?: NodeJS.WriteStream, stderr?: NodeJS.WriteStream, refreshRate?: number);
+    format: (message: string, indent: number) => string;

```

Comment:
Could you please follow @edwardfoyle 's suggestion from https://github.com/aws-amplify/amplify-backend/pull/952#discussion_r1468129526 ? I think his proposal is good and will scale better in the future when we add more "formatting options".

Reply:
didn't add the ones with color might be better to add them as part of https://github.com/aws-amplify/amplify-backend/pull/905